app/models/publication.rb

```diff
@@ -225,21 +225,32 @@ def year
   end
```

Comment:
A change from `String#include?` to an equality checker has some functional significance.  That might require analysis of the prod-db to be sure this change will be OK.  Other than that, I'm :+1: for it.  But it's a reserved/qualified :+1: until I can be sure that the prod-db is OK with it.  There's a lot of legacy data in there and all the code analysis in the world is not going to help me now whether or not this works with our prod-db.

Reply:
See https://github.com/sul-dlss/sul_pub/issues/625 for the analysis.  No problem with inexact matches, but there are 10 `nil` values (a separate issue).